packages/server/graphql/mutations/deleteUser.ts

```diff
@@ -65,13 +67,37 @@ export default {
     }
     const {id: userIdToDelete} = user
 
-    const deletedUserEmail = await softDeleteUser(
-      userIdToDelete,
-      dataLoader,
-      authToken,
-      validReason
-    )
+    const deletedUserEmail = await softDeleteUser(userIdToDelete, dataLoader)
     await markUserSoftDeleted(userIdToDelete, deletedUserEmail, validReason)
+
+    // Update HubSpot after deletion
+    const executeGraphQL = require('../executeGraphQL').default
+    const parabolPayload = await executeGraphQL({
```

Comment:
-1 you're now missing the event for `hardDeleteUser` mutation which is calling `softDeleteUser` as well.